src/main/resources/templates/request_reservation.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html xmlns:th="www.thymeleaf.org">
+<head>
+<meta charset="ISO-8859-1">
+<title>Request Reservation</title>
+<img src="/kanaAirlines.png" alt="Kana Airlines">
+</head>
+<body>
```

Comment:
Will expand this section to have selectable flights in sprint 2 once we have populated testing data in order.